.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -23,6 +23,8 @@
 ## Breaking changes
 <!--- List any breaking changes here. When the PR is merged please add an entry to https://github.com/AvaloniaUI/Avalonia/wiki/Breaking-Changes -->
 
+## Obsoletions / Deprications
```

Comment:
Not a native speaker, but shouldn't this be "deprecations"?

Reply:
fixed